front-end/src/components/Pages/Welcome.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import "../../styles/WelcomeRef.css";
+import Employees from "./Employees";
+import Sidebar from "../Parts/SideBar";
+
+
+function NewEmp() {
+  return (
+    <div className="new-employee-card">
+      <form action="">
+        Name: <input type="text" name="name" placeholder="full name" /><br />
```

Comment:
instead of just a full name, I would separate them into First Middle Last to have the search by name functionality available later.

Reply:
good idea let me implament that real quick